sdks/python/apache_beam/ml/inference/huggingface_inference.py

```diff
@@ -593,9 +594,11 @@ def __init__(
     Args:
       task (str or enum.Enum): task supported by HuggingFace Pipelines.
         Accepts a string task or an enum.Enum from PipelineTask.
-      model : path to pretrained model on Hugging Face Models Hub to use custom
-        model for the chosen task. If the model already defines the task then
-        no need to specify the task parameter.
+      model : path to the pretrained model-id on Hugging Face Models Hub
+        to use custom model for the chosen task. If the model already defines
+        the task then no need to specify the task parameter. Use the model_id
+        string instead of an actual model here. Model specific kwargs can be
+        specified with model_kwargs using load_pipeline_args.
```

Comment:
for my education, is it done like:  `load_pipeline_args={model_kwargs={key='value'}}`. or `model_kwargs` is passed directly into `load_pipeline_args` ? If you think it makes sense, you could add a snippet, like  "... for example `load_pipeline_args={model_kwargs={key='value'}}`"

Reply:
it would be the first one `load_pipeline_args={model_kwargs={key='value'}}`. added an example